src/Microsoft.AspNetCore.CertificateGeneration.Task/build/Microsoft.AspNetCore.CertificateGeneration.Task.targets

```diff
@@ -0,0 +1,17 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+********************************************************************************************
+Target: GenerateSSLCertificate
+
+Generates an SSL certificate to use for development in ASP.NET Core applications
+********************************************************************************************
+-->
+  <Target Name="GenerateSSLCertificate">
+    <PropertyGroup>
+      <ExecutablePath>$(MSBuildThisFileDirectory)..\tools\netcoreapp2.0\Microsoft.AspNetCore.CertificateGeneration.Task.dll</ExecutablePath>
+      <_Force Condition="$(ForceGenerateSSLCertificate) == 'true'">--force</_Force>
+    </PropertyGroup>
+    <Exec Command="dotnet $(ExecutablePath) ssl $(_Force)" />
```

Comment:
Two comments here:
```dotnet```
Anything that relies on PATH and is going to be fragile. Let's open an issue to track https://github.com/Microsoft/msbuild/issues/1669 for a better solution here. This is basically the same issue as https://github.com/aspnet/MvcPrecompilation/issues/50.

**MSBuild warnings/errors:**

If you don't take my suggestions above about prefixing console output, you can set `IgnoreStandardErrorWarningFormat="false"` here.